docs/devel/operator/lifecycle.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Lifecycle
+sidebar_position: 10
+description: >
+  Lilfecycle of an operator
+---
+
+This section describes the different operations that can be implemented by an
+operator and what is their use case.
+
+## Priority
+
+Operators are initialized (PreStart and Start) according to their priority and
+they are closed (Stop, PreStop and Close) in the reverse order.
+
+## Lifecycle
+
+### Init
```

Comment:
This name clashes with `Init` from `DataOperator` - can we use something that makes it easily distinguishable whether it is an operator or an operator instance?